template/apps/web/src/theme/main-theme.ts

```diff
@@ -1,13 +1,17 @@
-import { MantineTheme, MantineThemeOverride } from '@mantine/core';
+import { createTheme } from '@mantine/core';
 
-const mainTheme: MantineThemeOverride = {
+import components from './components';
+
+const mainTheme = createTheme({
   fontFamily: 'Roboto, sans-serif',
   fontFamilyMonospace: 'monospace',
   headings: {
     fontFamily: 'Roboto, sans-serif',
-    fontWeight: 600,
+    fontWeight: '600',
+  },
+  lineHeights: {
+    md: '1.45',
   },
-  lineHeight: 1.45,
   primaryColor: 'blue',
   primaryShade: 6,
   other: {
```

Comment:
let's remove`transition` data from `other` key, we can use Mantine [Transition](https://mantine.dev/core/transition/) component for creating transitions